presto-tpch/src/main/java/io/prestosql/plugin/tpch/TpchMetadata.java

```diff
@@ -153,6 +153,12 @@ private static StatisticsEstimator createStatisticsEstimator()
         return new StatisticsEstimator(tableStatisticsDataRepository);
     }
 
+    @Override
+    public boolean schemaExists(ConnectorSession session, String schemaName)
+    {
+        return schemaNameToScaleFactor(schemaName) < 0 ? false : true;
```

Comment:
TPCH has a bug -- `io.prestosql.plugin.tpch.TpchMetadata#getTableHandle` accepts scale factor 0, and so
```
select count(*) from tpch.sf0.nation;
```
- should fail with schema not found
- but it fails with:
```
java.lang.IllegalArgumentException: Scale factor must be larger than 0
	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:141)
	at io.prestosql.plugin.tpch.TpchTableHandle.<init>(TpchTableHandle.java:46)
	at io.prestosql.plugin.tpch.TpchTableHandle.<init>(TpchTableHandle.java:36)
	at io.prestosql.plugin.tpch.TpchMetadata.getTableHandle(TpchMetadata.java:176)
```

So:

- fix io.prestosql.plugin.tpch.TpchMetadata#getTableHandle's `if (scaleFactor ` as it is in tpcds
- make this line the same as in tpcds